### ISSUE TITLE
Help a noob to get this thing going #2: remote installation method

### ISSUE THREAD

**Issue by IanHeywood** [ratt-ru/radiopadre#38]
Following the build from source method, on the remote machine:

```
*snip*
install-radiopadre: Installation successful!
```

then on the local machine:

```
run-radiopadre --virtual-env --remote-path=/home/heywoodl/Software/radiopadre-devel heywoodl@aslxdaq04:/data/disk2/meerkat/thunderkat/GRS/radiopadre-default.ipynb
```

gives me:

```
un-radiopadre: Opening initial master connection to heywoodl@aslxdaq04. You may be prompted for your password.
run-radiopadre: Found remote radiopadre launch script at heywoodl@aslxdaq04:/home/heywoodl/Software/radiopadre-devel/bin/run-radiopadre
run-radiopadre: Testing the remote virtual environment at heywoodl@aslxdaq04:~/.radiopadre/venv
run-radiopadre: looks fine
heywoodl@aslxdaq04: DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
heywoodl@aslxdaq04: Traceback (most recent call last):
heywoodl@aslxdaq04: File "/home/heywoodl/.radiopadre/venv/bin/jupyter-notebook", line 6, in <module>
heywoodl@aslxdaq04: from notebook.notebookapp import main
heywoodl@aslxdaq04: File "/home/heywoodl/.radiopadre/venv/local/lib/python2.7/site-packages/notebook/__init__.py", line 25, in <module>
heywoodl@aslxdaq04: from .nbextensions import install_nbextension
heywoodl@aslxdaq04: File "/home/heywoodl/.radiopadre/venv/local/lib/python2.7/site-packages/notebook/nbextensions.py", line 31, in <module>
heywoodl@aslxdaq04: from .config_manager import BaseJSONConfigManager
heywoodl@aslxdaq04: File "/home/heywoodl/.radiopadre/venv/local/lib/python2.7/site-packages/notebook/config_manager.py", line 14, in <module>
heywoodl@aslxdaq04: from six import PY3
heywoodl@aslxdaq04: ImportError: No module named six
```

I did `source ~/.radiopadre/venv/bin/activate` then `pip install six`, which told me the requirement was already satisfied.

**Comment by Mulan-94** [comment]
@IanHeywood I speculate that this issue may be solved by solving #39   with the given suggestion. Do let us know....

**Comment by o-smirnov** [comment]
Yeah this may be tied to #39. @IanHeywood would you please try the branch and let me know?
